docs/JLBP-0015.md

```diff
@@ -23,10 +23,14 @@ in their own `<dependencyManagement>` section and omit the
 versions from the specific modules they import. The versions will
 be set by the BOM. This provides consistent versions for artifacts released
 by a single project.
-Your project's BOM should include every non-test-scoped module in the project.
+Your project's BOM shall include only those non-test-scoped modules in *your* project.
 Otherwise, a consumer who depends on an unspecified module must specify and
 upgrade the version of that module, defeating the purpose of the BOM.
 
+Your project's BOM shall not include any of your dependencies on other libraries.
+If your BOM specifies versions of a third party library then that will lead to BOM dependency 
```

Comment:
How about "Specifying versions of libraries from other projects can lead to BOM dependency conflicts that are exceedingly tricky to resolve"?